### DISCUSSION TITLE
[Module Manager] Proposal to improve usability

### DISCUSSION THREAD

**Discussion by vnt** [PrestaShop/PrestaShop#35174] [Ideas]
### The problem

Hi everyone,
It is not easy to quickly view which modules are active or not in the Module Manager.
I would like to propose to integrate a checkbox that allows you to see the status of the module and quickly activate or deactivate it.

### The solution

Specifically, I would suggest adding a checkbox next to each module name. This checkbox would be checked if the module is active, and unchecked if it is inactive.
This would make it much easier to quickly see the status of all of the modules. It would also make it easier to activate or deactivate modules, as you would simply need to click the checkbox.
I believe that this would be a valuable addition to the Module Manager. I hope that you will consider my proposal.
Thank you for your time.
David

![Screenshot 2024-01-25 alle 14 47 35](https://github.com/PrestaShop/PrestaShop/assets/1482194/0e28a91a-d008-4862-a19a-39e0e166106e)

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [X] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by MatShir** [comment]
![image](https://github.com/PrestaShop/PrestaShop/assets/43613217/a67a9668-257d-4412-9bf2-7f9c94794066)

here another design suggestion

**Reply by kpodemski** [reply]
Now that we don't have Enable/Disable mobile it would be great to have such look of the modules page :)

If only we could implement all ideas from @TristanLDD and @ElodieOS... 😅

**Reply by MatShir** [reply]
cc https://github.com/PrestaShop/PrestaShop/issues/27427#issuecomment-1027807335

**Comment by ElodieOS** [comment]
Hi everyone!
That's an excellent idea @vnt, thanks for sharing. I think it could be a good thing to simplify the enable/disable state of a module. Why not take the opportunity to rework the user experience of this page !
 
Using a toggle is a good recommendation, but it raises a few questions, particularly about the management of other functions. 

I'll look into it and keep you posted on progress :)
